extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/DereferenceAliasesMode.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.guacamole.auth.ldap;
+
+import com.novell.ldap.LDAPSearchConstraints;
+
+/**
+ * Data type that handles acceptable values for configuring
+ * alias dereferencing behavior when querying LDAP servers.
+ */
+public enum DereferenceAliasesMode {
+
+    /**
+     * Never dereference aliases.  This is the default.
```

Comment:
I'm guessing you were referring to spaces at the end of the sentences?  I've reduced those all to one - once upon a time it was considered correct to have two spaces, and I guess I revert to that every now and then, at the cost of being consistent about it.